.gitlab/common.yml

```diff
@@ -283,12 +328,12 @@ variables:
     - if [ "$cpe" = "true" ]; then
         echo "/usr/local/sbin/integration_mpi/exchange_domain skipped because cpe=$cpe";
         echo "/usr/local/sbin/integration_mpi/domain_nranks skipped because cpe=$cpe";
-        echo "/usr/local/bin/evrard skipped";
+        echo "/usr/local/bin/evrard_new skipped";
```

Comment:
I think this was renamed to `evrard`.

Reply:
https://github.com/unibas-dmi-hpc/SPH-EXA/pull/136/commits/5e571ad754b944402762ca27d92e1b150a34f19a